index.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="style.css">
+  <title>Learn something</title>
+</head>
+
+<body>
+  <!-- HERO VID -->
+  <div class="video-container">
+    <video autoplay loop muted>
+        <source src="dog.mp4" type="video/mp4">
+        Your browser does not support the video tag.
+    </video>
+    <div class="overlay-text">
+      <h1>YO!</h1>
+    </div>
+ </div>
+
+  <!-- HERO IMAGE -->
+  <!-- <header>
+  <div class="hero-image">
+    <div class="hero-text">
+      <h1>Hello</h1>
+      <p>Wanna Learn Something?</p>
+    </div>
+  </header> -->
+
```

Comment:
The code that we don't use can be deleted before making the pull request! If you want to save it you can make a commit with it and then it will be easy to get the code later on 😀